src/app/organizations/collection/add-entry/add-entry.component.html

```diff
@@ -33,10 +33,10 @@ <h1 mat-dialog-title>Add to collection</h1>
     </div>
   </app-loading>
 </div>
-<div mat-dialog-actions align="end">
+
+<div mat-dialog-actions fxLayout="row" fxLayoutAlign="flex-end center" fxLayoutGap="10px">
```

Comment:
https://github.com/angular/flex-layout/wiki/Declarative-API-Overview just use 'end' like the doc says, my editor doesn't even recognize flex-end as valid